QUEST_LV_0100.tsv

```diff
@@ -6208,7 +6208,7 @@ QUEST_LV_0100_20150317_006203	Defeated Rikaus that was spreading evil aura. Repo
 QUEST_LV_0100_20150317_006204	Defeat Rikaus
 QUEST_LV_0100_20150317_006205	Talk to Believer Evaldas
 QUEST_LV_0100_20150317_006206	$Goddess Saule Believer Evaldas needs your help.
-QUEST_LV_0100_20150317_006207	$Defeat the Merog Shamans
+QUEST_LV_0100_20150317_006207	$Defeat the Merlog Shamans
```

Comment:
Most of the game text uses Merlog apparently, should I change every other line to Merog instead?